functions/src/errorChecks.ts

```diff
@@ -0,0 +1,69 @@
+export function checkCVEValidity(ID: any): boolean {
+    const regex = /^CVE-\d{4}-\d{3,7}$/;
+    if (!regex.test(ID)) {
+      const hyphenString = ID.match(/-/g);
+      if (hyphenString) {
+        const hyphenCount = hyphenString.length;
+        const idCVEArray = ID.split("-");
+        if (hyphenCount === 3) {
+          if (idCVEArray[3].length === 1) {
+            return true;
+          }
```

Comment:
This block is a little fuzzy. I remembered that we only use the -1, -2, ... suffixes in patch analysis. I don't think we had them in bulletins anywhere. In this case, we could just remove this block.